pkg/optimizely/cache.go

```diff
@@ -35,13 +39,11 @@ import (
 	"github.com/optimizely/go-sdk/pkg/event"
 	"github.com/optimizely/go-sdk/pkg/logging"
 	"github.com/optimizely/go-sdk/pkg/odp"
+	odpCachePkg "github.com/optimizely/go-sdk/pkg/odp/cache"
```

Comment:
/v2 here too? 

Ok I'll stop tagging this point 😁

Reply:
Yes, seems v2 is very costly in Go :)

Reply:
Well, the cost is the confusion that non-Go devs cause 🤣  in asking so many basic questions.

Reply:
no no, not about the questions, lot of things are needed to update for /v2 changes :), seems it is good to stay under v2 😁